frappe/website/js/web_form.js

```diff
@@ -90,13 +90,9 @@ frappe.ready(function() {
 			callback: function(data) {
 				if(!data.exc) {
 					frappe.doc_name = data.message;
-					if(!frappe.login_required) {
-						show_success_message();
-					}
 
-					if(frappe.is_new && frappe.login_required) {
```

Comment:
why delete this?

Reply:
For the success message to be shown whenever a new web form is submitted, even if the user is not logged in.